src/quest_manager/models.py

```diff
@@ -19,12 +19,47 @@
 from tags.models import TagsModelMixin
 
 
+class CategoryManager(models.Manager):
+    """
+    Custom manager for Category model that provides
+    queryset methods for active campaigns with available quests.
+    """
+
+    def all_active_with_importable_quests(self):
+        """
+        Returns a queryset of active campaigns (categories) that have
+        at least one quest visible to students and not archived.
+
+        This helps efficiently filter campaigns relevant for display
+        or import in the library context.
+        """
+
+        # Start with all categories filtered to only those marked active
+        qs = self.get_queryset()
+
+        # Annotate each category with a count of related quests that:
+        # - are visible to students (not drafts)
```

Comment:
boom, not drafts, exactly!  I am realizing how inconstant we are throughout this project.  Campaign.active, Quest.visible_to_students , Badge.active, Announcement.draft

These should all be the same field name  * CRY *